apps/bfd-pipeline/bfd-pipeline-app/src/main/assembly/bin/bfd-pipeline-app.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+bfdPipelineAppJar="${scriptDirectory}/$(ls ${scriptDirectory} | grep '^bfd-pipeline-app-.*jar')"
```

Comment:
`bfdPipelineAppJar="${scriptDirectory}/bfd-pipeline-app-*.jar"` should work the same